internal/provider/sourcecontrol/resource_test.go

```diff
@@ -125,7 +125,7 @@ func TestMain(m *testing.M) {
 	resource.TestMain(m)
 }
 
-func TestAccSourceControl(t *testing.T) {
+func TestAccSourceControlTest(t *testing.T) {
```

Comment:
Had to add `...Test` in the end - otherwise, this test's name is a prefix for `TestAccSourceControlSettings`, and I couldn't run this test in isolation - `go test` would always run both, which is annoying.